### ISSUE TITLE
NullPointerException when class contains static final field with null value

### ISSUE THREAD

**Issue by lindmik** [jqno/equalsverifier#116]
I just updated EqualsVerifier from version 1.5 to version 1.7.2, and it now fails on a class that has a constant field with null value. The verification fails with the following stack trace:

```
java.lang.AssertionError: java.lang.NullPointerException: 
For more information, go to: http://www.jqno.nl/equalsverifier/errormessages
  at nl.jqno.equalsverifier.EqualsVerifier.handleError(EqualsVerifier.java:414)
  at nl.jqno.equalsverifier.EqualsVerifier.verify(EqualsVerifier.java:400)
  at ...
Caused by: java.lang.NullPointerException
  at nl.jqno.equalsverifier.FieldsChecker$ReflexivityFieldCheck.checkValueReflexivity(FieldsChecker.java:306)
  at nl.jqno.equalsverifier.FieldsChecker$ReflexivityFieldCheck.execute(FieldsChecker.java:282)
  at nl.jqno.equalsverifier.FieldInspector.check(FieldInspector.java:37)
  at nl.jqno.equalsverifier.FieldsChecker.check(FieldsChecker.java:62)
  at nl.jqno.equalsverifier.EqualsVerifier.verifyWithExamples(EqualsVerifier.java:460)
  at nl.jqno.equalsverifier.EqualsVerifier.performVerification(EqualsVerifier.java:427)
  at nl.jqno.equalsverifier.EqualsVerifier.verify(EqualsVerifier.java:394)
  ... 24 more
```

**Comment by jqno** [comment]
Nice catch, thanks for reporting it. I'll fix it in the next version.

**Comment by jqno** [comment]
I've just released version 1.7.3, which fixes this.
